### DISCUSSION TITLE
Initial support for Wi-Fi extender

### DISCUSSION THREAD

**Discussion by sergiu46** [arendst/Tasmota#13464] [Q&A] [answered]
What this feature means? Can we make esp devices with tasmota also wi-fi extenders?

**Comment by Jason2866** [comment, marked as answer]
Take a look in the docu https://tasmota.github.io/docs/Range-Extender/

**Reply by sergiu46** [reply]
This is what i was asking for. Thanks for replay. Do you know if in the next release this will be included in precompiled binaries?

**Reply by Jason2866** [reply]
Not in official builds but there are the unofficial ones. Provided from the Tasmota team.
https://github.com/tasmota/install/tree/main/firmware/unofficial
Installed via https://tasmota.github.io/install/

**Comment by sfromis** [comment]
Not clear exactly what you refer to.
https://tasmota.github.io/docs/Range-Extender/
https://github.com/arendst/Tasmota/blob/development/info/xdrv_57_tasmesh.md

**Comment by sergiu46** [comment]
I have some esp8266 devices with weak signal and this could help me
